samples/azure/azure-service-fabric-routing/sample.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Azure Service Fabric Partition Aware Routing
+reviewed: 2017-02-14
+component: Core
+related:
+- nservicebus/azure-service-bus
+---
+
+
+## Prerequisites
+
+1. Strong understanding of Service Fabric [Reliable Services](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-quick-start). 
+1. Azure Service Fabric [dev cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started) running and configured with 5 nodes.
+1. An Azure Service Bus namespace that can be used for communication between the instances.
+1. A system environment variable named "AzureServiceBus.ConnectionString" set to the connection string of the Azure Service Bus namespace. The connection string must provide Manage rights.
+
+Note: A Service Fabric cluster runs under the Network Service account and only reads system environment variables. Make sure the environment variable "AzureServiceBus.ConnectionString" is defined as a system environment variable and not user-scoped.
+
+
+## Scenario
+
+The scenario used in this sample covers a voting system. In this voting system the cast votes are counted by candidate. The cast votes are `published` from the voter client to the endpoint responsible for counting candidate votes.
+
+Next to this the system also counts the total number of votes cast in each zip code. In order to achieve this the candidate voting endpoint issues a `request` to the zip code counting endpoint to track the zip code. The zip code counting endpoint will `reply` back with the intermediary results.
+
+When the election is closed, the candidate vote counting endpoint will `publish` the results per candidate and report them using Service Fabric diagnostics infrastructure (ETW Event Viewer windows).
```

Comment:
this seems to be the first time we reference "Service Fabric diagnostics infrastructure (ETW Event Viewer windows)"

perhaps add a link to more info?